nodecg-io-core/dashboard/esbuild.config.js

```diff
@@ -0,0 +1,107 @@
+// @ts-check
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+/* eslint-disable no-console */
+
+const esbuild = require("esbuild");
+const path = require("path");
+const process = require("process");
+const fs = require("fs");
+
+const args = process.argv.slice(2);
+const prod = process.env.NODE_ENV === "production";
+
+const entryPoints = [
+    "monaco-editor/esm/vs/language/json/json.worker.js",
+    "monaco-editor/esm/vs/language/css/css.worker.js",
+    "monaco-editor/esm/vs/language/html/html.worker.js",
+    "monaco-editor/esm/vs/language/typescript/ts.worker.js",
+    "monaco-editor/esm/vs/editor/editor.worker.js",
+    "main.ts",
+];
```

Comment:
We only need the main editor and the json module of monaco. The rest can be safely removed as we're not using monaco to edit css, html or javascript.
This reduces the build output from 48,6 MiB to 21.0 MiB or from 8.2 MiB to 3.6 MiB if gzipped (as it is in tarballs/the npm registry).
```suggestion
const entryPoints = [
    "monaco-editor/esm/vs/language/json/json.worker.js",
    "monaco-editor/esm/vs/editor/editor.worker.js",
    "main.ts",
];
```